src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -40,9 +40,9 @@
 
 FILENAME_LENGTH = 255
 LOG_NAME_REPLACE_STR = "##NAME##"
-DEFAULT_PROCESS_TIMEOUT = 5400
-DEFAULT_SUITE_TIMEOUT = 5400
-DEFAULT_TEST_TIMEOUT = 5400
+DEFAULT_PROCESS_TIMEOUT = 3600
```

Comment:
@wangleis @dmitry-gorokhov 
I will ask you not to do such changes in such utils without review from ie test infra maintainers. Such changes can affect GHA CI
CC @andrei-kochin

Reply:
@iefode [PR 22489](https://github.com/openvinotoolkit/openvino/pull/22489) is reverted in this PR and replaced by [PR 23099](https://github.com/openvinotoolkit/openvino/pull/23099) which did not change GHA CI setting. Thanks.

Reply:
FYI: this script is used to run func tests in GHA. It is the main reason why I asked you do not make such changes without approve